Obsidian/Net/MinecraftStream.Writing.cs

```diff
@@ -345,7 +347,21 @@ public void WriteFloatAngle(Angle angle)
     public async Task WriteAngleAsync(Angle angle)
     {
         await WriteByteAsync((sbyte)angle.Value);
-        // await this.WriteUnsignedByteAsync((byte)(angle / Angle.MaxValue * byte.MaxValue));
+    }
+
+    [WriteMethod]
+    public void WriteBitSet(BitSet bitset, bool isFixed = false)
+    {
+        //TODO WE HAVE TO DO SOMETHING ABOUT THIS
```

Comment:
Right now it's assuming the length of the bitset is 1 and writing a byte but ideally we should make should sure the bit array has a inner byte array and write that instead of writing the long array if isFixed is true. It's fine for now though